examples/google-alloydb-chat-message-history-example/google-alloydb-chat-message-history-example.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/memory/alloydb"
+	"github.com/tmc/langchaingo/util/alloydbutil"
+)
+
+// getEnvVariables loads the necessary environment variables for the AlloyDB connection
+// and the chat message history creation.
+func getEnvVariables() (string, string, string, string, string, string, string, string, string) {
```

Comment:
The chat package needs to update the JSON -> JSONB (https://github.com/averikitsch/langchaingo/blob/61c9f41fe2166d6f907ebcb136f69ab929da24e1/memory/alloydb/chat_message_history.go#L77)
I'm getting error:
"error validating table 'chat_me' in schema 'public': error, column 'data' in table 'chat_me' has type 'jsonb', but expected type 'json'"

Reply:
done! just fixed it and verified locally, example runs for me after that!